extensions/wordpress-docs/src/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { useState } from "react";
+import { XMLParser } from "fast-xml-parser";
+import { useFetch, Response } from "@raycast/utils";
+import { ActionPanel, Action, List, Icon } from "@raycast/api";
+
+export default function Command() {
+  const [searchText, setSearchText] = useState("");
+
+  const { data, isLoading } = useFetch(`https://developer.wordpress.org/search/${searchText}/feed/rss/`, {
+    parseResponse: parseFetchResponse,
+    execute: searchText.length > 0,
+  });
+
+  return (
+    <List
+      isLoading={isLoading}
+      onSearchTextChange={setSearchText}
+      searchBarPlaceholder="Search WordPress code reference..."
+      throttle
+    >
+      {searchText.length > 0 ? (
+        <List.Section title="Results" subtitle={data?.length + ""}>
+          {data?.map((searchResult: SearchResult) => (
+            <SearchListItem key={searchResult.name} searchResult={searchResult} />
+          ))}
+        </List.Section>
+      ) : (
+        <List.EmptyView title={"Type to search"} />
```

Comment:
being greeted by an empty view is pretty daunting 😅 

the Laravel Docs extension greets users with 
<img width="851" alt="image" src="https://user-images.githubusercontent.com/1155589/232084232-fc4eeee9-5513-4249-a00f-8254d34e76b4.png">

and Tailwind CSS with
<img width="955" alt="image" src="https://user-images.githubusercontent.com/1155589/232084366-1584c1c6-595a-48f1-b34f-f3efccf941f3.png">

What about making this extension greet users with something like the content of the Wordpress docs homepage:
<img width="1942" alt="image" src="https://user-images.githubusercontent.com/1155589/232084574-cba0034e-c54a-4808-ac46-df8d3ed31374.png">

Reply:
Added in https://github.com/raycast/extensions/pull/5833/commits/b3497e47ab70d93e730c316453a0d5a2a4c88ccb 👍